src/components/AllCreaturesTile.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const AllCreaturesTile = props => {
+    // const
```

Comment:
if you have to define a const here please do so, otherwise please remove the commented out code